frontend/src/lib/components/LemonActionableTooltip/LemonActionableTooltip.scss

```diff
@@ -0,0 +1,43 @@
+.LemonActionableTooltip {
+    max-width: var(--in-app-prompts-width);
+    .LemonActionableTooltip__header {
+        display: flex;
```

Comment:
**not blocking** but makes me wonder about our custom CSS metric... should we be extending utility classes in files like this 🤔

Reply:
and that's why something like tailwind was useful :D

I think one future goal should be to make custom classes just a combination of utility classes